qbft/spectest/all_tests.go

```diff
@@ -101,14 +101,14 @@ var AllTests = []SpecTest{
 	proposer.ThirteenOperators(),
 
 	messages.RoundChangeDataInvalidJustifications(),
-	messages.RoundChangeDataInvalidPreparedRound(),
-	messages.RoundChangeDataInvalidPreparedValue(),
+	// TODO<olegshmuelov>: rc not prepared with data
+	//messages.RoundChangeDataInvalidPreparedRound(),
```

Comment:
failing test should be resolved
https://github.com/bloxapp/ssv-spec/pull/69/files#r1023970249

Reply:
https://github.com/bloxapp/ssv-spec/issues/103
could be related